### PR TITLE
Fix/remove v2 features

### LINKED ISSUE #96: Remove version 2 features from develop-v1 branch
develop-v1 should feature an early static version of the website.

### PR DESCRIPTION
**Pull request by Fabricevladimir**
Commented out and removed certain v2 features. Closes #96

### CODE REVIEW

**Comment by 0xLeif** [comment]
should this be merged into develop-v1?

**Comment by Fabricevladimir** [comment]
> should this be merged into develop-v1?

Hehehe. Awesome catch.

File: src/components/Toolbar/HamburgerToolbar.jsx [hunk 1]
```diff
@@ -40,26 +36,15 @@ function HamburgerToolbar() {
...
-              <NavLink to="/join">Join</NavLink>
-            </li>
+          <ul data-testid='link-dropdown' className='link-dropdown'>
             <li>
-              <NavLink to="/about-us">About</NavLink>
+              <NavLink to='/contact-us'>Active Projects</NavLink>
             </li>
             <li>
-              <NavLink to="/posts">Posts</NavLink>
+              <NavLink to='/active-projects'>Contact Us</NavLink>
```
Comment (will-nemo) [inline src/components/Toolbar/HamburgerToolbar.jsx]: Looks like the links are mixed up with their labels.

Should be:

 ```<NavLink to='/active-projects'>Contact Us</NavLink>```

File: src/components/Toolbar/__tests__/Toolbar.test.jsx [hunk 2]
```diff
@@ -1,36 +1,24 @@
...
+      </UserProvider>
+    );
+
+    // 4 links (home, contact us, active projects, meet the team)
+    expect(queryAllByRole('link').length).toBe(4);
+    expect(queryByText('Contact Us')).toBeInTheDocument();
+    expect(queryByText('Active Projects')).toBeInTheDocument();
+    expect(queryByText('Meet the Team')).toBeInTheDocument();
+  });
+});
```
Comment (will-nemo) [inline src/components/Toolbar/__tests__/Toolbar.test.jsx]: Should we be deleting this these tests sine we will likely want them in V2? Can we just adds skips to them for now?

https://stackoverflow.com/questions/48125230/skip-one-test-in-test-file-jest

File: src/style-sheets/layout/_hamburger-toolbar.scss [hunk 3]
```diff
@@ -42,6 +42,7 @@
       width: 100%;
 
       & div.search-box-wrapper {
+        display: none; // v-2
```
Comment (will-nemo) [inline src/style-sheets/layout/_hamburger-toolbar.scss]: Can you add TODO's wherever we're hiding things for v2? Would help us find things faster and keeps things scoped

Comment (Fabricevladimir) [reply src/style-sheets/layout/_hamburger-toolbar.scss]: Sure thing!

File: src/components/Toolbar/__tests__/Toolbar.test.jsx (hunk 2 above)
Comment (Fabricevladimir) [reply src/components/Toolbar/__tests__/Toolbar.test.jsx]: We can do that.

**Pull request merged**